k8s/helm-chart/dynatrace-gcp-function/templates/template.yml

```diff
@@ -228,4 +228,13 @@ spec:
             path: /health
             port: 8081
           initialDelaySeconds: 10
+      {{- if (eq .Values.activeGate.useExisting "false") }}
+      initContainers:
+        - name: init-active-gate-service
+          image: busybox:1.33.1
+          command: [ 'sh', '-c', "until nslookup dynatrace-activegate-gcpmon-clusterip.dynatrace.svc.cluster.local; do echo waiting for AG service; sleep 2; done" ]
```

Comment:
if you are on the same namespace then use short link: 
```
until nslookup dynatrace-activegate-gcpmon-clusterip
```
instead of:
```
until nslookup dynatrace-activegate-gcpmon-clusterip.dynatrace.svc.cluster.local
```

Reply:
Good to know :)